http-server-tck/src/main/java/io/micronaut/http/server/tck/tests/MiscTest.java

```diff
@@ -158,6 +160,16 @@ void applicationJsonWithBodyAnnotationAndObjectReturnRenderedAsJson() throws IOE
                 .build()));
     }
 
+    @Test
```

Comment:
Can you move the test into a separate file?. Having so many tests in a single file is a mistake. I caused that. Having them in individual files allow us to exclude them more granularly.

Reply:
Good call 👍  Done in https://github.com/micronaut-projects/micronaut-core/pull/8712/commits/130433a9b866a3ecf8d4359e70910b1fbdeb0121